.github/workflows/test.yml

```
@@ -47,12 +47,18 @@ jobs:
       - run: npm run lint
       - run: npm run markdownlint
       - run: npm run test:unit
-      - shell: bash
+
+      - name: Run npm run test:e2e
+        shell: bash
         run: |
+          curl https://github.com/VOICEVOX/voicevox_engine/releases/download/0.14.4/voicevox_engine-windows-cpu-0.14.4.7z.001 -fOsSL
+          7z x voicevox_engine-windows-cpu-0.14.4.7z.001
+          mv windows-cpu voicevox_engine
+          cp .env.test .env
+          sed -i -e 's|"../voicevox_engine/run.exe"|"'$(pwd | sed -Ee 's_^/([a-z])/_\1:/_')'/voicevox_engine/run.exe"|' .env
```

絶対パスにしてるんですね。たぶんしなくても動くはず？
```suggestion
          sed -i -e 's|"../voicevox_engine/run.exe"|"voicevox_engine/run.exe"|' .env
```

---

しないと動きませんでした。上の方で失敗してるはず

---

なるほどです。